trade-finance-manager-api/src/v1/helpers/create-tasks-amendment.helper.js

```diff
@@ -323,7 +323,7 @@ const updateAmendmentTasks = async (facilityId, amendmentId, taskUpdate) => {
  *                        a 'groupTasks' key with a list of tasks.
  * @param {string} groupName - The name of the team/group which tasks should be assigned.
  * @param {string} newUserId - The ID of the user to whom tasks should be assigned.
- * @returns {Array} A modified list of task groups.
+ * @returns {Promise<Array>} A modified list of task groups.
```

Comment:
I believe the generic argument for the `Array` interface is required, so this should either be `Promise<object[]>` to match the above, or `Promise<Array<object>>`